### PR TITLE
doc: Add Pulse Connect Secure intake for OC

### PR DESCRIPTION
**Pull request by otetard**

### CODE REVIEW

**Comment by otetard** [comment]
@celinequivy Can you have a quick look at this draft please? The Pulse configuration is missing (users should configure their Pulse appliance to forward its logs to the intermediate Rsyslog, that will then forward them to SEKOIA.IO)

File: docs/integrations/pulse.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+$ActionSendStreamDriver gtls # use gtls netstream driver
+$ActionSendStreamDriverMode 1 # require TLS for the connection
+$ActionSendStreamDriverAuthMode x509/name # server is authenticated
+
+# Template definition [RFC5424](https://tools.ietf.org/html/rfc5424#section-7.2.2)
+# IMPORTANT: don't forget to set your intake key in the template
+template(name="SEKOIAIOPulseTemplate" type="string" string="<%pri%>1 %timestamp:::date-rfc3339% %hostname% %app-name% %procid% LOG [SEKOIA@53288 intake_key=\"YOUR_INTAKE_KEY\"] %msg%\n")
+
+# Send your events to SEKOIA.IO intake servers under SEKOIAIOPulseTemplate template
+if $programname startswith 'named' then @@(o)intake.sekoia.io:10514;SEKOIAIOPulseTemplate
```
Comment (celinequivy) [inline docs/integrations/pulse.md:43]: Is 'named' the real name, or is it a variable ?

**Review by celinequivy — APPROVED** [review]

File: docs/integrations/pulse.md (hunk 1 above)
Comment (otetard) [reply docs/integrations/pulse.md:43]: Oups, wrong copy/paste :)

Comment (otetard) [reply docs/integrations/pulse.md]: fixed

**Review by celinequivy — APPROVED** [review]

**Pull request merged**